lib/compress/zstd_double_fast.c

```diff
@@ -298,6 +298,14 @@ size_t ZSTD_compressBlock_doubleFast_dictMatchState_generic(
     /* if a dictionary is attached, it must be within window range */
     assert(ms->window.dictLimit + (1U << cParams->windowLog) >= endIndex);
 
+    /* when the CDict is assumed cold, we prefetch its tables */
+    if (ms->prefetchCDictTables) {
+        size_t const hashTableSize = ((size_t)1) << dictCParams->hashLog;
```

Comment:
note: I'm using `dictCParams->hashLog` directly because the short cache PR changes the definition of `dictHBitsL` etc